lib/charms/vault_k8s/v0/vault_client.py

```diff
@@ -169,6 +169,17 @@ def is_sealed(self) -> bool:
             logger.error("Error while checking Vault seal status: %s", e)
             raise VaultClientError(e) from e
 
+    def is_available_initialized_and_unsealed(self) -> bool:
+        """Return whether Vault is available, initialized and unsealed.
+
+        It surpasses intermittent errors when checking seal status.
```

Comment:
```suggestion
        It surpasses intermittent errors when checking seal status.
```

I am not sure the word `surpasses` is right here. My guess is that it prevents those intermittent errors, but I am not sure. Can you please reword this?

Reply:
I'm not sure how to. But I ended up writing exactly what happens. (In case of intermittent error we return False)